bin/create.js

```diff
@@ -37,6 +50,10 @@ var tokenOptions = {
         'default': 'default',
         type: 'string',
     },
+    auth0: {
+        description: 'Authenticate webtask with Auth0: --auth0 <clientID> <client-secret> <auth0-domain>',
```

Comment:
They were previously separate options, but since you have to specify all three together anyway I thought they'd be better rolled into one. Will make them comma separated.